src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -263,6 +264,21 @@ public Visitable visit(PackageDeclaration node, Void arg) {
     return super.visit(node, arg);
   }
 
+  @Override
+  public Node visit(ImportDeclaration decl, Void arg) {
+    @SuppressWarnings(
+        "signature") // since this is the content of an import statement, it will be a qualified
+    // class path
+    @FullyQualifiedName String declAsString = decl.getNameAsString();
+    // if there is no jar paths included, every imported class is unsolved, apart from those
+    // belonging to the Java language.
+    if (!classesFromJar.isEmpty() || declAsString.startsWith("java")) {
```

Comment:
Users are also allowed to add classes to the `java` namespace. For example, consider a program with this import:
```
import javaproject.MyClass;
```

This is totally valid (and I've seen students do similar things plenty of times...), but this check will misclassify such an import as solved.